python/python-video-generation/code/Pipfile

```diff
@@ -11,6 +11,7 @@ autopep8 = "*"
 [packages]
 moviepy = "==1.0.3"
 scipy = "==1.10.1"
+pillow = "9.5.0"
```

Comment:
If this is the specific version we need, please add it like

```sh
pillow = "==9.5.0"
```

The current will install the latest version that match "9.5.*"